M/MUMPS_seq/build_tarballs.jl

```diff
@@ -43,7 +25,7 @@ make_args+=(OPTF=-O
             CC="$CC -fPIC ${CFLAGS[@]}"
             FC="gfortran -fPIC ${FFLAGS[@]}"
             FL="gfortran -fPIC"
-            LIBBLAS=${OPENBLAS})
+            LIBBLAS="-L${prefix}/lib -lopenblas")
```

Comment:
@giordano Since this is failing on windows, should it be ${libdir}?

Reply:
I wonder how come it was working with libopenblas64_.

Reply:
Yes, using `${libdir}` is definitely a good idea in any case